filesystem/path.h

```diff
@@ -11,7 +11,7 @@
 
 #include "fwd.h"
 #include <string>
-#include <vector>
+#include <vector>r
```

Comment:
Hi -- please test your code before submitting a pull request. If it breaks the other project, it is not that helpful ... ;)